composer.json

```diff
@@ -24,14 +24,14 @@
         "phpunit/phpunit": "^11.5.0"
     },
     "autoload": {
-        "classmap": [
-            "src/"
-        ]
+        "psr-4": {
+            "Lcobucci\\JWT\\": "src"
+        }
```

Comment:
I doubt this can work given the `jwt` package has the same `autoload` section.
I would keep the `classmap` here

Reply:
The fact the tests do work proves it does work.

The generated `vendor/composer/autoload_psr4.php` has these lines:

`    'Lcobucci\\JWT\\Tests\\' => array($baseDir . '/tests'),`
`    'Lcobucci\\JWT\\' => array($baseDir . '/src', $vendorDir . '/lcobucci/jwt/src'),`

There's basically only one alternative: Being more specific in the namespace prefix, e.g. go with `Lcobucci\JWT\Signer\RsaPss` (and move the base class file "RsaPss.php").

Reason for me avoiding classmaps as the original definition is that this imposes some sort of premature optimization. Another argument (very weak here, as the scope is narrowly defined, i.e. not much new code is expected in the future) is that classmaps are inherently cumbersome to develop new code with, as it requires regularly updating the autoloader.

Reply:
Let's hope this works in production too :crossed_fingers: